src/main/resources/static/_shared.yml

```diff
@@ -3392,6 +3392,16 @@ components:
         - rotl
         - release_following_decision
         - additional_placement
+    WithdrawPlacementApplication:
+      type: object
+      properties:
+        reason:
+          type: string
+          enum:
+            - DuplicatePlacementRequest
+            - AlternativeProvisionIdentified
```

Comment:
Same here vis a vis the named object for the enum. Could potentially share it with the `WithdrawPlacementRequest` object too

Reply:
Good idea! have done this